23-07-workshop/in-progress/query_region_plot_HR_plot_lc.ipynb

```diff
@@ -0,0 +1,1368 @@
+{
```

Comment:
<p>Choose a well-known cluster, motivate age determination</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Decided against this - covered in an Exercise</p>